awx/ui/client/features/output/details.component.js

```diff
@@ -126,6 +126,26 @@ function getSourceWorkflowJobDetails () {
     return { link, tooltip };
 }
 
+function getShardDetails () {
+    const internalLimitDetails = resource.model.get('summary_fields.internal_limit');
+
+    if (!internalLimitDetails) {
+        return null;
+    }
+
+    const shardDetails = resource.model.get('summary_fields.internal_limit.shard');
+
+    if (!shardDetails) {
+        return null;
+    }
+
+    const label = strings.get('labels.SHARD_DETAILS');
+    const offset = `${shardDetails.offset} of ${shardDetails.step} shards`;
```

Comment:
This is currently using a zero-based value for the offset:
![image](https://user-images.githubusercontent.com/28987580/45972253-1fb7de00-bff0-11e8-9253-8dd4d520e42c.png)

Reply:
Yes that is correct. It's trivial to +1 to the value, but as of now, I'm unsure if I want to do that on the API side or the UI side. Right now I'm favoring the UI side because these are the parameters we take in the inventory script endpoint. For these kind of aesthetic things, I would like to audit all of the name changes and implement them all at once. The tower-qa tests will also need to be changed at the point.

Reply:
@one-t just wanted to loop you in - alan and I talked this one out at the office. Current plan is to center it to `1` as you've suggested.